src/Sparrow/Sparrow.csproj

```diff
@@ -23,6 +24,13 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'Validate' ">
     <DefineConstants>$(DefineConstants);MEM_GUARD_STACK</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard1.3|AnyCPU'">
+    <DefineConstants>TRACE;RELEASE;NETSTANDARD1_3</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Validate|netstandard1.3|AnyCPU'">
+    <DefineConstants>TRACE;RELEASE;NETSTANDARD1_3;VALIDATE;</DefineConstants>
```

Comment:
IIRC you can use:
<DefineConstants>$(DefineConstants);VALIDATE;</DefineConstants>

This also will be applied only to netstandard1.3

Reply:
Apparently it copied it from the Release which has:
``` 
<DefineConstants>TRACE;RELEASE;NETSTANDARD1_3</DefineConstants>
```